Vagrantfile

```diff
@@ -43,7 +43,7 @@ Vagrant.configure("2") do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  # config.vm.synced_folder "../data", "/vagrant_data"
+  config.vm.synced_folder ".", "/vagrant"
```

Comment:
Isn't that the default?

Reply:
Not on this VM (VMWare) at least - it wasn't mounting anything by default.